lib/Mojo/WebSocketProxy/Backend/ConsumerGroups.pm

```diff
@@ -338,9 +340,9 @@ sub _prepare_request_data {
         [
         rpc      => $method,
         args     => encode_json_utf8($params),
```

Comment:
This is conceptually wrong, the *entire* set of parameters needs to be UTF8-encoded. Doing partial encoding like this tends to lead to problems later if someone adds another key.

Reply:
It going to be added to the stream by Redis `XADD` command. Since it accepts the even scalar parameters, so we have to encode arrays and hashes only. Actually, encoding at the moment is not a good idea unless we going to pass all of the parameters only in a pair of params to `XADD`.